runtime/doc/treesitter.txt

```diff
@@ -353,6 +353,34 @@ attribute: >
     )
 <
 
+Treesitter Spell Highlighting                            *lua-treesitter-spell*
+
+This is completely integrated with Neovim's built-in spellchecker, and
```

Comment:
What is "this"? You could start with a short(!) paragraph explaining the _purpose_ of this module (and maybe how it works), then how to set spell regions, and then mention that the rest behaves like default spelling (`see |spell|`).

Reply:
I copied the syntax highlighting section which also begins with "this".

Reply:
We can do better :) 

(vim.treesitter docs generally need some love; that's a dedicated item on the tracking issue...)

Reply:
Ok, I've had another go at re-wording.

Reply:
Yep, that's better!